frontend/src/components/Forms/CoffeeChatsForm/CoffeeChatsDashboard.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Card, Loader, Message, Button } from 'semantic-ui-react';
+import styles from './CoffeeChatsForm.module.css';
+import CoffeeChatAPI from '../../../API/CoffeeChatAPI';
+import { Emitters } from '../../../utils';
+
+const CoffeeChatsDashboard = (props: {
+  approvedChats: CoffeeChat[];
+  pendingChats: CoffeeChat[];
+  rejectedChats: CoffeeChat[];
+  isChatLoading: boolean;
+  setPendingChats: Dispatch<SetStateAction<CoffeeChat[]>>;
+}): JSX.Element => {
+  const { approvedChats, pendingChats, rejectedChats, isChatLoading, setPendingChats } = props;
+
+  const deleteCoffeeChatRequest = (coffeeChat: CoffeeChat) => {
+    CoffeeChatAPI.deleteCoffeeChat(coffeeChat.uuid)
+      .then(() => {
+        setPendingChats(pendingChats.filter((currChat) => currChat.uuid !== coffeeChat.uuid));
+        Emitters.generalSuccess.emit({
+          headerMsg: 'Coffee Chat Deleted!',
+          contentMsg: 'Your coffee chat was successfully deleted!'
+        });
+        Emitters.coffeeChatsUpdated.emit();
+      })
+      .catch((error) => {
+        Emitters.generalError.emit({
+          headerMsg: 'You are not allowed to delete this coffee chat!',
+          contentMsg: error
+        });
+      });
+  };
+
+  const CoffeeChatsDisplay = (props: { coffeeChatList: CoffeeChat[] }): JSX.Element => {
+    const { coffeeChatList } = props;
+
+    return (
+      <Card.Group>
+        {coffeeChatList.map((coffeeChat) => (
+          <Card key={coffeeChat.uuid}>
+            <Card.Content>
+              <Card.Header>Coffee Chat with {coffeeChat.otherMember.firstName}</Card.Header>
+
+              <div
+                style={{
+                  display: 'flex',
+                  justifyContent: 'space-between',
+                  alignItems: 'center',
+                  marginTop: '10px'
+                }}
+              >
+                <div>
+                  <Card.Meta>Category: {coffeeChat.slackLink}</Card.Meta>
+                  <Card.Meta>
+                    <b>
+                      <a href={coffeeChat.slackLink}>Image Link</a>
```

Comment:
```suggestion
                      <a href={coffeeChat.slackLink} target="_blank">Image Link</a>
```
can you make this open up in a new tab?